RELEASING.md

```diff
@@ -1,32 +1,8 @@
-#### (Somewhat) automated process: 
+#### Releasing: 
 1. Start a git-flow release/x.y.z
 1. Create a CHANGELOG.latest.md with the changes for the current 
 version (to be used by Fastlane for the github release notes)
 1. Update the version number by running `bundle exec fastlane bump_and_update_changelog version:x.y.z`
```

Comment:
Could we also automate this? The way I see it is:

1. We open a PR against develop with branch name `release/x.y.z`, we modify the CHANGELOG.latest.md, or do an empty commit so that we can open the PR
1. CircleCI runs this command and pushes a commit to the PR we just opened. I am pretty sure we can detect the version number from the branch name
1. Merge PR to develop, update main (do we really need main?), push a new tag
1. CircleCI finishes release

Reply:
that would be pretty cool. I'd do it as a separate PR, though, since this already kills the step where we deploy, and that would kill the step where we make the bumps